### PR TITLE
Drop DomF from the product

### PR DESCRIPTION
**Pull request by iartemenko**

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend [hunk 1]
```diff
@@ -31,7 +31,7 @@ do_install_append_r8a7795() {
         sed -e '$a\\' \
             -e '$a\[output]' \
             -e '$a\name=HDMI-A-1' \
-            -e '$a\transform=270' \
+            -e '$a\transform=0' \
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend]: Do we need to do the same for r8a7796?

Comment (arminn) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend]: According to prod-devel requirements we need to support M3.

Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend]: Updated

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg [hunk 2]
```diff
@@ -47,7 +47,7 @@ vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 # connector 1001 - second android connector on AGL application display (1088x1487):
 #     * AGL provides layout (x=0, y=218, w=1080, h=1487) for HMI application;
 #     * we use 1088 to be aligned with tile 32 (pixels), it will be scaled to 1080 by DisplayManager.
-vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:768x1024' ]
+vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:1024x768' ]
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg]: What about other config files? doma-salvator-x-h3-4x2g.cfg for example.

Comment (otyshchenko1) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg]: Also I think that you should to correct comment above "vdispl =" string. Seems it is not relevant.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-h3.cfg [hunk 3]
```diff
@@ -8,17 +8,16 @@ displays = (
...
 );
-
 layers = (
     {
         name = "agl";
         id = 1001;
         display = "agl";
-        width = 1080;
-        height = 1487;
+        width = 1024;
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-h3.cfg]: Should the DM configuration be changed for M3 as well?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg (hunk 2 above)
Comment (iartemenko) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg]: Updated

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-h3.cfg (hunk 3 above)
Comment (iartemenko) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-h3.cfg]: Updated

**Comment by iartemenko** [comment]
Any comments here?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg (hunk 2 above)
Comment (aanisov) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg]: But the comment above the vdispl still does refer AGL which gone away long ago. And describes outdated displays layout.

**Comment by aanisov** [comment]
Also, I would like to see more informative PR titles and/or descriptions in the future. "Use 3 domain instead of four" is pretty abstract. IMHO, something like "Drop DomF from the product" would attract more attention to your PR.

**Comment by iartemenko** [comment]
Done

**Pull request merged**